lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -8,9 +8,22 @@ module Overcommit::Hook::PreCommit
   # somewhere else. Example:
   #
   #   EsLint:
+  #     required_executable: 'npm'
   #     enabled: true
   #     command: ['npm', 'run', 'lint']
   #
+  # And currently this hook supports only eslint compact format, you can configure
+  # this in package.json. Example:
```

Comment:
You should also be able to configure it for this hook without changing your lint script, like this:

```yaml
command: ['npm', 'run', 'lint', '--', '-f', 'compact']
```

I haven't tested this, but I think it will work.